axelrod/strategies/titfortat.py

```diff
@@ -636,3 +636,31 @@ def strategy(self, opponent: Player) -> Action:
 
         # Otherwise play previous move
         return self.history[-1]
+
+@FinalTransformer(D, name_prefix=None)
```

Comment:
`FinalTransformer` takes a sequence. Change this to `(D, )` (although that doesn't fix what's going wrong).